lib/src/gutter/error.dart

```diff
@@ -1,14 +1,173 @@
+    final color = theme.styles['root']?.color;
+    final style = widget.style.copyWith(
+      fontSize: 14,
+      color: color,
+      backgroundColor: backgroundColor,
+      fontStyle: FontStyle.normal,
+    );
+    return MouseRegion(
+      onEnter: (event) {
+        setState(() {
+          showErrorDetails = true;
+          enteredPopup = false;
+          if (entry != null) {
+            return;
+          }
+          entry = getErrorPopup(
+            widget.issue,
+            offset: event.position.translate(-5, -5),
+            style: style,
+          );
+          final overlay = Overlay.of(context);
+          overlay?.insert(entry!);
+          overlay?.build(context);
+        });
+      },
+      onExit: (event) {
+        Future.delayed(
+          const Duration(milliseconds: 50),
+          () {
+            setState(() {
+              showErrorDetails = false;
+              if (!enteredPopup) {
+                entry?.remove();
+                entry = null;
+              }
+            });
+          },
+        );
+      },
+      child: icon,
+    );
+  }
+
+  OverlayEntry getErrorPopup(
+    Issue issue, {
+    required Offset offset,
+    required TextStyle style,
+  }) {
+    final renderBox = context.findRenderObject() as RenderBox?;
+    final width = renderBox?.size.width ?? 16;
+    final newOffset = renderBox?.localToGlobal(Offset.zero) ?? offset;
+    return OverlayEntry(
+      builder: (context) {
+        return Positioned(
+          left: newOffset.dx + width,
+          top: newOffset.dy,
+          child: MouseRegion(
+            onEnter: (event) => setState(() {
+              enteredPopup = true;
+            }),
+            onExit: (event) => setState(() {
+              entry?.remove();
+              entry = null;
+            }),
+            child: DefaultTextStyle(
+              style: style,
+              child: Row(
+                mainAxisSize: MainAxisSize.min,
+                children: [
+                  Container(
+                    padding: const EdgeInsets.all(12),
+                    constraints: const BoxConstraints(
+                      minWidth: 100,
+                      maxWidth: 500,
+                    ),
+                    decoration: BoxDecoration(
+                      color: style.backgroundColor,
+                      border: Border.all(
+                        color: style.color ??
+                            const Color.fromARGB(107, 255, 255, 255),
+                      ),
+                    ),
+                    child: Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      mainAxisSize: MainAxisSize.min,
+                      children: [
+                        Text(
+                          issue.message,
+                        ),
+                        if (issue.url != null) ...[
+                          Text(issue.url!),
```

Comment:
Make it clickable.

Reply:
I didn't find any default link opener in flutter. For now I am making a call to `html.window.open(url)`
Should I create a dependency to `url_launcher` and use that instead?